phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java

```diff
@@ -40,4 +44,23 @@ public void testUnknownOption() {
         exit.expectSystemExitWithStatus(1);
         Pherf.main(args);
     }
+
```

Comment:
I don't see a test for bad values of this passed in, non-integer negative etc.

Reply:
sad path added